src/components/cron.ts

```diff
@@ -126,3 +129,28 @@ export const createCoffeeChatCron = (client: Client): CronJob =>
       throw 'Bad channel type';
     }
   });
+
+export const assignCodeyRoleForLeaderboard = (client: Client): CronJob =>
+  new CronJob('0 0 * * * *', async function () {
+    const leaderboard = await getCoinLeaderboard(10);
+    const guild = client.guilds.resolve(TARGET_GUILD_ID);
+    if (!guild) {
+      throw 'guild not found';
```

Comment:
@mcpenguin So I can do that, but I am not sure if this will be caught in codeyCommand because there is no message to reply to. 
My codeyUserError works on replying to a message with the error, but here there is no initial message. Not sure how I can make it work.

Reply:
Crons are not run by the main codeyCommand framework, they are handled separately by another library. So these errors would be logged in console, and anyways the guild not existing should be a breaking bug.